lib/Mojo/Cookie/Response.pm

```diff
@@ -123,6 +126,14 @@ HttpOnly flag, which can prevent client-side scripts from accessing this cookie.
 
 Max age for cookie.
 
+=head2 partitioned
+
+  my $partitioned = $cookie->partitioned;
+  $cookie         = $cookie->parititoned(1);
+
+Partitioned flag value on all session cookies, this is to be used in accordance to the L<CHIPS ammendment
+to RFC 6265|https://www.ietf.org/archive/id/draft-cutler-httpbis-partitioned-cookies-00.html>.
```

Comment:
I don't quite understand why session cookies are mentioned here specifically.

Reply:
I had the documentation backwards my bad. This should be on the sessions pod not the Cookie::Response pod.